apps/main/src/loan/components/ChartOhlcWrapper/LiquidityData.tsx

```diff
@@ -88,7 +87,9 @@ const LiquidityData: React.FC<LiqudityDataProps> = ({ llammaControllerData, chai
             )}
           </LiquidityEvent>
           <TimestampColumn>
-            <Tooltip tooltip={`${convertTime(transaction.timestamp)} ${convertFullTime(transaction.timestamp)}`}>
+            <Tooltip
+              tooltip={`${convertTime(transaction.timestamp)} ${convertDate(transaction.timestamp).toLocaleDateString()}`}
```

Comment:
I think `toLocaleDateString` is the right one to call, however it would be nice to keep it in a helper method so it's easy to change

Reply:
Made a note to actually put the entire tooltip message in a helper method